radicle-cli/examples/workflow/3-issues.md

```diff
@@ -46,6 +47,20 @@ $ rad issue list --assigned
 ╰─────────────────────────────────────────────────────────────────────────────────────────────────────╯
 ```
 
+In addition, you can see that when you run `rad issue show` the issue add you to `Assignees`.
+
+```
+$ rad issue show b05e945
```

Comment:
It's weird that the output form the `rad issue list --assigned` command above doesn't show `You` but this one does. I think this is what @cloudhead mentioned that we need to standardise how this is done otherwise it will sometimes show `You` and sometimes show your `NID`, and maybe even sometimes show your `DID`.

Probably worth discussing how this should be done before making everything drift apart visually :)

Reply:
for now I restore the `did`